docs/zkapps/how-to-deploy-a-zkapp.mdx

```diff
@@ -1,6 +1,14 @@
 ---
 title: How to Deploy a zkApp
 hide_title: true
+description: A step-by-step guide on how to deploy a zkApp to the Berkeley Testnet. The guide covers configuring your smart contract project, requesting funds from the faucet, and deploying your smart contract.
+keywords:
+  - zkApp
+  - Deployment
+  - Berkeley Testnet
```

Comment:
```suggestion
```

Reply:
oh hmmm, do we want to put a testnet name into the keywords? In this case, the topic covers the steps to deploy to Berkley Testnet... we'll need to remove all of the Berkley Testnet mentions after hard fork, right? (what comes after testnet? mainnet?)

Reply:
Good catch! Lets not deal with the headache of removing it later and just do it now! :D